.github/setup-unit.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Setup script for Kestra Odoo Plugin Unit Tests
+# This script sets up a local Odoo instance for testing both locally and in CI
+
+set -e
+
+echo "🐳 Setting up Odoo for unit tests..."
+
+# Check if Docker and Docker Compose are available
+if ! command -v docker &> /dev/null; then
+    echo "❌ Docker is not installed or not in PATH"
+    exit 1
+fi
+
+if ! command -v docker-compose &> /dev/null; then
+    echo "❌ Docker Compose is not installed or not in PATH"
+    exit 1
+fi
+
+# Stop and remove any existing containers
+echo "🧹 Cleaning up existing containers..."
+docker-compose -f odoo-docker-compose.yml down -v --remove-orphans || true
```

Comment:
Could you rename the file to just `docker-compose.yml` please?

```suggestion
docker-compose -f docker-compose.yml down -v --remove-orphans || true
```